tailwind.config.js

```diff
@@ -0,0 +1,11 @@
+module.exports = {
+  content: [
+    './src/**/*.{js,jsx,ts,tsx}',
+    './node_modules/tw-elements/dist/js/**/*.js',
+  ],
+  theme: {
+    extend: {},
+  },
+  // eslint-disable-next-line global-require
```

Comment:
what is this comment about ?

Reply:
I followed Tailwind Elements docs.
![image](https://user-images.githubusercontent.com/44370085/161967227-554ebf6f-caf9-4b13-a57f-5f20124ccc94.png)
Prettier give me a warning about global require and I searched about that warning but I couldn't solve it. Just for temporary I used like that.
There is one solution but didn't work for me.
![image](https://user-images.githubusercontent.com/44370085/161967905-3dc3949e-4080-4498-83b7-1fc64cb605fc.png)
Lastly, If I don't use plugin part, Tailwind Elements doesn't work.

Reply:
There is an eslint exception on that comment, we try not to add exceptions so the plugin part is okay, let us delete that comment since it does not do anything. After your response about the accordion width, let us merge this branch and resolve the issue.

PS: Please do not forget updating your branch so we wont have many merge conflicts.